Makefile

```diff
@@ -21,3 +21,11 @@ deps:
 build: deps
 	go build .
 	@echo >&2 executable saved to ./gcosmos
+
+.PHONY: install
+install: deps
+	go install
```

Comment:
I will take a closer look at the whole PR later, but let's stick with just make build so we don't have to worry about anything on $PATH.  Then have the new scripts use ./gcosmos instead of path.

Reply:
It cleans up the demo to use `gcosmos` instead of `./gcosmos` and makes it feel more "real" I think. But it does prevent footguns with inexperienced go users who have not added their GOBIN to their PATH to use the local bin. I'm pretty torn on which is better.

Reply:
I get what you're saying but it's a safer assumption that the user only has write access to the current directory. It also avoids the possibility of path weirdness where they had an outdated build earlier in their PATH and then we have to troubleshoot that.